skglm/cv.py

```diff
@@ -0,0 +1,139 @@
+import numpy as np
+from joblib import Parallel, delayed
+from skglm.datafits import Logistic, QuadraticSVC
+from skglm.estimators import GeneralizedLinearEstimator
+
+
+def _kfold_split(n_samples, k, rng):
```

Comment:
can you rely on sklearn KFold() instead ? It uses stratified kfold by default and should yield shorter code (less error prone)